packages/web-react/config/rollup.config.js

```diff
@@ -31,17 +31,13 @@ function isExternal(id, parentId, entryPointsAreExternal = true) {
   if (path.isAbsolute(id)) {
     const posixId = toPosixPath(id);
     const posixParentId = toPosixPath(parentId);
-    absoluteId = path.posix.relative(
-      path.posix.dirname(posixParentId),
-      posixId,
-    );
+    absoluteId = path.posix.relative(path.posix.dirname(posixParentId), posixId);
```

Comment:
Hmm, I'd like to get back to this at some point, this really isn't more readable than before.

Reply:
This is the same problem with print width.

Reply:
Maybe we should stay with 80 for js files.